extensions/blocks/podcast-player/style.scss

```diff
@@ -445,5 +446,15 @@ $player-background: transparent;
 		.mejs-unmute > button {
 			mask-position: -40px 0;
 		}
+
+		.jetpack-podcast-player__track-status-icon--playing {
+			mask-position: -60px 4px;
+			max-width: 20px;
+			max-height: 24px;
```

Comment:
Can we use the icon size variable here? Also, can we update that variable to 24px (currently 22px) to match the error icon? (Related issue: https://github.com/Automattic/jetpack/issues/15373)

Reply:
> Can we use the icon size variable here?
We can for the height. The width needs to be lower otherwise the next icon in the sprite starts showing.

> Also, can we update that variable to 24px (currently 22px) to match the error icon? (Related issue: #15373)

The goal of this PR is to address https://github.com/Automattic/jetpack/issues/15337. I'd like to keep it to a singular change. If we want to update the variable to 24 pixel, we should do that in a PR addressing https://github.com/Automattic/jetpack/issues/15373. Also, there's some discussion to be had whether https://github.com/Automattic/jetpack/issues/15373 should be an issue. While most SVG icons in Gutenberg have a canvas of 24x24, the actual visual icon sizes vary quite a bit between 17-24px. Currently, our two icons are visually more or less the same size. Given that I'm not sure I see the benefit of changing their container to 24px.

Reply:
> We can for the height. The width needs to be lower otherwise the next icon in the sprite starts showing.

👍 

> The goal of this PR is to address #15337. I'd like to keep it to a singular change. If we want to update the variable to 24 pixel, we should do that in a PR addressing #15373.

Absolutely! I only mentioned the 24px issue because you set the size of the new `playing` icon to 24px (see the line we're commenting on in this thread) which would _almost_ close the #15373, so it made sense to me to set 24px for error icon too and feed two birds with one scone. Since you've updated the `playing` icon to use the 22px variable instead of a fixed 24px value, I'm happy to move the 24px issue to another PR. Thanks!